crowbar_framework/app/models/nova_service.rb

```diff
@@ -184,7 +184,18 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
 
     neutron = ProposalObject.find_proposal("neutron",role.default_attributes["nova"]["neutron_instance"])
 
-    all_nodes.each do |n|
+    compute_nodes_for_network = []
+    role.override_attributes["nova"]["elements"].each do |role, nodes|
+      # only care about compute nodes
+      next unless role =~ /^nova-multi-compute-/
+      # vmware compute nodes do not need access to the networking
+      next if role == "nova-multi-compute-vmware"
```

Comment:
Why wouldn't they need it? Or let me rephrase: they don't really need it because they only support vlan for now, but I don't think we should keep this restriction in the long term and instead use ovs for windows.

The rationale I used is really: is the compute node just a proxy to something else, or is it actually hosting stuff?